src/safe-upgrade-tool/run.md

```diff
@@ -0,0 +1,143 @@
+---
+group: software-update-guide
+title: Run the tool
+ee_only: True
+functional_areas:
+  - Upgrade
+---
+
+The Magento Safe Upgrade Tool ALPHA (SUT) is a command line (CLI) tool that checks a Magento instance against a specific version by analyzing all the non-Magento modules installed on it.
+
+The SUT identifies potential problems that must be fixed in your custom code before attempting to upgrade to a newer version of Magento.
+
+The tool returns a list of errors and warnings that you must address before upgrading to a new version of Magento.
+
+{:.bs-callout-warning}
+At the moment this is an ALPHA version with limited scope, only validating PHP Magento APIs and GraphQL schema.
+
+## Use SUT
+
+### Memory limitations
+
+If you are running a system with large third-party modules and files, the SUT requires a high amount of RAM, at least 2GB RAM.
+
+### Executing the tool
```

Comment:
On this area "Executing the tool" we must add the -m command as it was developed by the team to execute the tool against a specific directory/module